service/controller/resource/scrapeconfigs/test/aws/case-0-cluster-api.golden

```diff
@@ -725,10 +799,18 @@
   relabel_configs:
   - source_labels: [__meta_kubernetes_service_annotationpresent_giantswarm_io_monitoring]
     regex: (true)
-    action: keep
-  - source_labels: [__meta_kubernetes_service_annotation_giantswarm_io_monitoring]
+    target_label: __tmp_keep_me
+    replacement: true
+  - source_labels: [__meta_kubernetes_service_labelpresent_giantswarm_io_monitoring]
+    regex: (true)
```

Comment:
I'd vote for only using the label `giantswarm.io/monitoring`, that's what we documented in the spec.

Reply:
We cannot as we need to support legacy apps that used this label

Reply:
What are those? Just asking as annotation is not compatible with selectors so this would block being able to use the _prometheus-operator_ CRs to manage things.

Reply:
I adjusted some apps but they are present in existing clusters still. Older releases will have them

Reply:
The annotations were selected by halo and batman before atlas even existed